README.md

```diff
@@ -47,7 +47,7 @@ Field Name               |  Description
 ------------------------ | -------------------------------------------------------------------------------------------- | ---------------
 AD LENGTH [MFG SPECIFIC] | Length of the type and data portion of the Manufacturer Specific advertising data structure. | `0x1B`
 AD TYPE [MFG SPECIFIC]   | Type representing the Manufacturer Specific advertising data structure.                      | `0xFF`
-MFG ID                   | The beacon device manufacturer's company identifier code.                                    | The little endian representation of the beacon device manufacturer's company code as maintained by the Bluetooth SIG assigned numbers database
+MFG ID                   | The beacon device manufacturer's company identifier code.                                    | The little endian representation of the [beacon device manufacturer's company code](http://www.bluetooth.com/specifications/assigned-numbers/company-Identifiers) as maintained by the Bluetooth SIG assigned numbers database
```

Comment:
It should actually be `[beacon device manufacturer's company identifier code](https://www.bluetooth.com/specifications/assigned-numbers/company-identifiers)` (note the "... company **identifier** code]..." and the more recent http**s** link using all-lowercase.